### PR TITLE
wayfire: 0.7.2 -> 0.7.3

### PR DESCRIPTION
**Pull request by Pacman99**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/window-managers/wayfire/wf-shell.nix [hunk 1]
```diff
@@ -15,7 +16,7 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ meson ninja pkg-config wayland ];
   buildInputs = [
-    alsa-lib gtkmm3 gtk-layer-shell pulseaudio wayfire wf-config
+    alsa-lib gtkmm3 gtk-layer-shell pulseaudio wayfire wf-config wlroots
```
Comment (Pacman99) [inline pkgs/applications/window-managers/wayfire/wf-shell.nix:19]: should `wlroots` be in propagatedBuildInputs for `wayfire`?

Comment (wucke13) [reply pkgs/applications/window-managers/wayfire/wf-shell.nix:19]: @Pacman99 I don't see an obvious reason why it should be in `propagatedBuildInputs`, why would you propose that?

**Comment by cole-h** [comment]
@Pacman99 Would you be willing to update to the latest version of wayfire (0.7.4)? Nothing terribly important (clarifies some licensing thing of one of the plugins), but would be good to get the latest released version :)

**Comment by stephaneyfx** [comment]
This seems superseded by #203754.

**Pull request closed without merging**